bin/generate-bundle-types

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
```

Comment:
I'm not 100% sold on the pattern here as it feels a bit surprising but went with something we did on a couple Dependabot projects. Here all you need to do is have docker installed and run `bin/generate-bundle-types` and should "just work".

I moved the curl downloads out from and set them up in the `Dockerfile` so we can run this bin script without network access 🔒